src/site/asn2.rst

```diff
@@ -54,8 +54,25 @@ Part 0 --- Read the Assignment
 Read the assignment description in its entirety before starting.
 
 
-Part 1 --- Read the Javadoc Reference
-=====================================
+Part 1 --- Read Through the Provided Code
+=========================================
+
+Read through all the provided code. The code is commented and there are method headers describing everything. These are
+objects and methods you will be using to solve the problem.
+
+Reading through and understanding existing code is a **huge** part of the assignment. Although there are no marks
+directly associated with it, it is a requirement in order to do anything useful. Further, if you continue studying CS,
+you will be reading documentation and other people's code **a lot**.
+
+Be sure to also read through all the test classes. Take the time to make sense of what is going on --- these will save
+you.
```

Comment:
May want to also tell them to read the junit docs for the few bits you use outside of `@Test`. `@Nested` and `@ParameterizedTest` may be confusing to the uninitiated.

Reply:
have a looksie at what I did